language/functional_tests/tests/testsuite/natives/vector/vector_move_after_destroy.mvir

```diff
@@ -0,0 +1,11 @@
+import 0x0.Vector;
+
+main() {
+    let v: Vector.T<u64>;
+    v = Vector.empty<u64>();
+    Vector.destroy_empty<u64>(move(v));
+    v = move(v);
+    return;
+}
+
+// check: MOVELOC_UNAVAILABLE_ERROR
```

Comment:
Is this test related to the implementation of `destroy_empty`? It seems like this is testing that you can't double-move a resource. I can't think of a bad implementation of `destroy_empty` that would make this fail. If that's the case, maybe we should get rid of this test.